Makefile

```diff
@@ -188,16 +187,19 @@ retina-capture-workload: ## build the Retina capture workload
 
 ##@ Containers
 
-RETINA_BUILDER_IMAGE = retina-builder
-RETINA_TOOLS_IMAGE = retina-tools
-RETINA_IMAGE = retina-agent
-RETINA_INIT_IMAGE = retina-init
-KUBECTL_RETINA_IMAGE=kubectl-retina
-RETINA_OPERATOR_IMAGE=retina-operator
-RETINA_INTEGRATION_TEST_IMAGE=retina-integration-test
-RETINA_PROTO_IMAGE=retina-proto-gen
-RETINA_GO_GEN_IMAGE=retina-go-gen
-KAPINGER_IMAGE = kapinger
+IMAGE_REGISTRY	?= ghcr.io
+IMAGE_NAMESPACE	?= $(git remote get-url origin | sed -e 's/git@github.com://g' -e 's/\.git$//') # attempts to extract the upstream for image namespacing
+
+RETINA_BUILDER_IMAGE			= $(IMAGE_NAMESPACE)/retina-builder
```

Comment:
This is more of a nit/opinion, but can we not have separate `IMAGE_NAMESPACE` and `IMAGE_RESGISTRY`?

My thinking is like this
```
IMAGE_REGISTRY ?= ghcr.io/$(git remote get-url origin | sed -e 's/git@github.com://g' -e 's/\.git$//')

RETINA_BUILDER_IMAGE = retina-builder
...
```
The benefit is any user who want's to use any other registry can update just the `IMAGE_REGISTRY` env variable and commands would simply work.

I think this will make pipelines simpler as well?

Reply:
wontfix in this change, we can evaluate simplifying the image name flow in the future